### DISCUSSION TITLE
Authentication failed

### DISCUSSION THREAD

**Discussion by jitendrahostmudcom** [community/community#53489] [Repositories] [answered]
### Select Topic Area

Bug

### Body

Hello All,

I am using Ubuntu OS.
I try to clone repository from Github but it is showing an error.

Cloning into '/home/jitendra/jitendra/python/FRCM-Website1'...
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Authentication failed for 'https://github.com/Progrow-Farm-Rural-Mission-Pvt-Ltd/FRCM-Website.git/'

Please check and guide me.

Thanks,
Jitendra

**Comment by airtower-luna** [comment, marked as answer]
As the message says, you cannot use your password for `git push` or other Git operations (but you still need it to log in). You need to [create a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) and enter that instead when Git asks for your password. When creating the token make sure to select the **repo** scope (for repository access, like pushing) and if you use Actions also **workflow** (required to modify workflow files).
